src/structures/FormTextInput.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+const BaseMessageComponent = require('./BaseMessageComponent');
+const { FormInputTextStyles } = require('../util/Constants');
+const Util = require('../util/Util');
+
+/**
+ * @typedef {'SHORT'|'PARAGRAPH'} FormInputTextStyle
```

Comment:
Not sure if this part is a WIP or not, but made it match the style of the other typedefs. Also please move it closer to where it's used for organization purposes :)
```suggestion
/**
 * The style of a form input's text field
 * * SHORT
 * * PARAGRAPH
 * @typedef {string} FormInputTextStyle
```